File: KoKoslan/src/kokoslan/java/exception/KoKoNotFoundId.java
/**
 @author loriacarlos@gmail.com
 
*/
package kokoslan.java.exception;

import kokoslan.java.ast.*;

public class KoKoNotFoundId extends RuntimeException {
	public KoKoNotFoundId(KoKoId id){
		super( String.format("*** Id '%s' is not defined! ***", id.getValue()) );
	}
	
}

File: KoKoslan/src/kokoslan/kotlin/exception/KoKoNotFoundId.kt
/*
    Andrey Arguedas Espinoza
    Daniela Armas Sanchez
    Michael Chen Wang
    Kimberly Olivas Delgado
*/

package kokoslan.kotlin.exception

import kokoslan.kotlin.ast.*

class KoKoNotFoundId(id: KoKoId) : RuntimeException("*** Id ${id.getValue()} is not defined! ***")